pom.xml

```diff
@@ -594,8 +595,6 @@ under the License.
                             <shadeTestJar>true</shadeTestJar>
                             <shadedArtifactAttached>false</shadedArtifactAttached>
                             <createDependencyReducedPom>true</createDependencyReducedPom>
-                            <dependencyReducedPomLocation>${project.basedir}/target/dependency-reduced-pom.xml
```

Comment:
Do we need a `maven-clean-plugin` like https://github.com/apache/flink/commit/60101e120fb2bd2393e9da00f07e1d53cec35c23

Reply:
> Do we need a `maven-clean-plugin` like [apache/flink@60101e1](https://github.com/apache/flink/commit/60101e120fb2bd2393e9da00f07e1d53cec35c23)

Done.